<?xml version="5.0"?>
<launch>

<arg name="rate" default="1.0" />
<arg name="meshfile" default="/media/amock/SanDiskAlex/datasets/mulran/KAIST/range_mcl_mesh_mulran_kaist_v3.ply" />
<arg name="bagfile" default="/media/amock/SanDiskAlex/datasets/micp_mulran/KAIST/mulran_kaist02_micp.bag" />


<node pkg="rosbag" type="play" name="rosbag" args="$(arg bagfile) -r $(arg rate)" output="screen" />

<node pkg="radarays_ros" type="radar_simulator" name="radar_simulator" output="screen">
    <param name="map_file" type="string" value="$(arg meshfile)" />
    <param name="map_frame" type="string" value="map" />
    <param name="sensor_frame" type="string" value="radar_polar" />
    <rosparam command="load" file="$(find radarays_ros)/config/mulran_kaist02.yaml" />
</node>

<node name="dynamic_reconfigure_load" pkg="dynamic_reconfigure" type="dynparam" args="load /radar_simulator $(find radarays_ros)/cfg/mulran_kaist_nonoise_dyncfg.yaml" />

</launch>
